pkg/artifactory/repositories.go

```diff
@@ -702,6 +721,16 @@ var baseRemoteRepoSchema = map[string]*schema.Schema{
 		Default:     false,
 		Description: `(Optional) Lists the items of remote folders in simple and list browsing. The remote content is cached according to the value of the 'Retrieval Cache Period'. Default value is 'false'.`,
 	},
+	"mismatching_mime_types_override_list": {
+		Type:             schema.TypeString,
+		Optional:         true,
+		ValidateDiagFunc: commaSeperatedList,
+		StateFunc: func(thing interface{}) string {
```

Comment:
Moved "mismatching_mime_types_override_list" field from npm remote repo resource. Since this field is seen in all remote repos, i moved it to generic field schema. 
Example usage: mismatching_mime_types_override_list = "application/json,application/xml"
Note: Original logic was from your PR #219 (4 months back). REST API takes this field as a string.

Reply:
yes, I remember writing this - and if I can't remember what was up, then it def requires some comments. I do, now, however, remember why I did this. It was so that I could treat the lists as equal even if the user input merely changes order. So, I sorted the fields and put them back together for state

Please add a description - `A comma seperated list ...` Try and find some official docs. When I was writing this even I wasn't being so consistent. But we have a much more active community now so please add some documentation. 

It's your code now. So, "Well, hey, that's how it was before" isn't really showing an ownership mentality

Reply:
Sorry ... My intend is not to indicate ownership. It is our code. I meant to say that i just moved.
There is no official docs on this field. 
Will add a description like you suggested.